src/common/Journald.h

```diff
@@ -0,0 +1,70 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#ifndef __CEPH_LOG_JOURNALD_H
+#define __CEPH_LOG_JOURNALD_H
+
+#include <memory>
+#include <sys/types.h>
+#include <sys/socket.h>
+
+struct LogEntry;
+
+namespace ceph {
+
+namespace logging {
+
+namespace detail {
+class EntryEncoder;
+class LogEntryEncoder;
+
+class JournaldClient {
+ public:
+  JournaldClient();
+  ~JournaldClient();
+  void send();
+  struct msghdr m_msghdr;
+ private:
+  int fd;
+};
+}
+
+class Entry;
+class SubsystemMap;
+
+class Journald {
```

Comment:
nit, might want to rename it to `JournaldLocalLogger`. as we don't implement `Journald` here. what we have here is a logger sending log messages to journald.

and add a brief comment to explain the difference between it and `JournaldClusterLog`.